.chloggen/parentbased-jaeger-remote.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+component: samplers
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add ParentBasedJaegerRemote sampler & validate argument
+
+# One or more tracking issues related to the change
+issues: [1801]
```

Comment:
1801 is this PR not issue (the issue doe not exist).

Reply:
chlog requires a value for `Issue`.  If no issue is present, an PR number should be used so that the changelog has something to link to.

Reply:
Also, GitHub redirects to PR if there is number is not an issue (ex: https://github.com/open-telemetry/opentelemetry-operator/issues/1801).